earth_enterprise/src/installer/install_server.sh

```diff
@@ -121,22 +538,13 @@ postgres_db_config()
   reset_pgdb
 }
 
-run_as_user() {
- use_su="su $1 -c 'echo -n 1' 2> /dev/null  || echo -n 0"
- if [ "$use_su" -eq 1 ] ; then
-    ( cd / ;su $1 -c "$2" )
- else
-    ( cd / ;sudo -u $1 $2 )
- fi
-}
-
 reset_pgdb()
 {
   # TODO check if correct
   # a) Always do an upgrade of the psql db
   echo 2 | run_as_user $GEPGUSER_NAME "/opt/google/bin/geresetpgdb upgrade"
   echo -e "upgrade done"
-  
+
```

Comment:
GEPGUSER_NAME appears to be a possible command line parameter (vs. GEFusionUser)